projects/imagemagick/crop_fuzzer.cc

```diff
@@ -0,0 +1,22 @@
+#include <cstdint>
+
+#include <Magick++/Blob.h>
+#include <Magick++/Image.h>
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size) {
+  if (Size < 2 * sizeof(uint16_t)) {
+    return 0;
+  }
+  size_t Width = *reinterpret_cast<const uint16_t *>(Data);
+  size_t Height = *reinterpret_cast<const uint16_t *>(Data + sizeof(Width));
```

Comment:
There is a type mismatch between `Data + sizeof(Width)` and the check in the beginning of the function, you may read out of bounds of the `Data` buffer on this line. I'd declare `Width` and `Height` in the beginning, than check the size against sum of `sizeof(Width) + sizeof(Height)`, then initialize the values.

Reply:
Fixed! (Whoops 😢)